shakenfist/daemons/net.py

```diff
@@ -94,12 +95,57 @@ def _maintain_networks(self):
         db.persist_node_vxid_mapping(
             config.parsed.get('NODE_NAME'), vxid_to_mac)
 
+    def _process_network_node_workitems(self):
+        workitem = db.dequeue('networknode')
+        if not workitem:
+            time.sleep(0.2)
+            return
+
+        if 'network_uuid' not in workitem:
+            return
```

Comment:
Probably should warn here since this would be a code bug

Reply:
Done.